nncf/openvino/graph/nncf_graph_builder.py

```diff
@@ -177,9 +178,16 @@ def create_nncf_graph(model: ov.Model) -> NNCFGraph:
                     if GraphConverter.convert_to_nncf_dtype(ov_dtype) == Dtype.INTEGER:
                         continue
 
+                    transpose = False
+                    if metatype == OVMatMulMetatype:
+                        attribute_names = ["transpose_a", "transpose_b"]
+                        node_attributes = node.get_attributes()
+                        transpose = node_attributes[attribute_names[const_port_id]]
+
                     const_attrs[const_port_id] = {
                         "name": const_node.get_friendly_name(),
                         "shape": tuple(const_node.get_output_shape(0)),
+                        "transpose": transpose,
```

Comment:
Can we add `transpose` attr only for `MatMul` ops?

Reply:
done